functions/src/index.ts

```diff
@@ -257,6 +382,57 @@ const generateReadingsCsvRuntimeOptions: functions.RuntimeOptions = {
   timeoutSeconds: 540,
 };
 
+/**
+ * This function populates the given sensor doc with a default circular buffer
+ * for eitherAQI or PM2.5
+ * @param aqiBuffer - true if AQI buffer, false if PM25 buffer
+ * @param docRef - document reference for the sensor to update
+ */
+async function populateBuffer(
+  aqiBuffer: boolean,
+  docRef: FirebaseFirestore.DocumentReference<FirebaseFirestore.DocumentData>
+) {
+  let bufferSize = 0;
```

Comment:
Since it looks like `bufferSize` is only defined and used within each if statement, don't define it here